source/game.database.bmx

```diff
@@ -1693,7 +1707,7 @@ Type TDatabaseLoader
 			"production_broadcast_flags", "production_licence_flags", "production_broadcast_limit", ..
 			"live_date", "live_time", ..
 			"broadcast_time_slot_start", "broadcast_time_slot_end", ..
-			"production_limit", "production_time", "production_time_min", "production_time_max", "production_time_slope", "production_time_mod" ..
+			"production_limit", "production_time_mod" ..
```

Comment:
Hmm ... weiss nicht, ob "production_time_mod" dann einfach in `<production_time>` untergebracht werden sollte? Oder in "data", weil es eher "generischerer" Natur ist?

Reply:
Aus "Sprachsicht" würde ich production_time_mod als "mod" in dem Tag nur unterbringen wollen, wenn ein mod-Wert auch für alle anderen value/min/max/slope-Werte unterstützt wird (was aber nicht überall sinnvoll ist). Zu überlegen wäre auch, ob man analog Programmen und News auch in scripttamplates Modifiers unterstützt und production_time_mod dann dorhin wandert.
Aktuell scheint der Wert nirgendwo in der Datenbank verwendet zu werden. Insofern könnte ich den Wert auch erst einmal aus der Dokumenation nehmen. Wenn dann geklärt ist, wo er final untergebracht werden soll, wird Implementierung und Dokumentation entsprechend angepasst.